public/app/features/admin/UserOrgs.tsx

```diff
@@ -224,9 +225,21 @@ export class AddToOrgModal extends PureComponent<AddToOrgModalProps, AddToOrgMod
     const { isOpen } = this.props;
     const { role } = this.state;
     const styles = getAddToOrgModalStyles();
-
+    const modalStyles = getModalStyles(this.props.theme);
+    const contentClassName = cx(
+      modalStyles.modalContent,
```

Comment:
This this is a left over from the previous solution, this is already applied in the Modal component

Reply:
Awesome catch, thank you!